internal/controllers/machinedeployment/machinedeployment_rolling.go

```diff
@@ -179,6 +179,12 @@ func (r *Reconciler) reconcileOldMachineSets(ctx context.Context, allMSs []*clus
 
 	log.V(4).Info("Cleaned up unhealthy replicas from old MachineSets", "count", cleanupCount)
 
+	log.V(4).Info("Ensure old MachineSet has deletion related timeouts propagated from MachineDeployment")
+	if err := r.patchOldMSDeletionTimeoutsFromMD(ctx, oldMSs, deployment); err != nil {
+		return err
+	}
+	log.V(4).Info("Ensured old MachineSet has deletion related timeouts propagated from MachineDeployment")
+
```

Comment:
```suggestion
	if err := r.patchOldMSDeletionTimeoutsFromMD(ctx, oldMSs, deployment); err != nil {
		return err
	}
```

I think this log doesn't have information for debugging.